docs/book/deployment.md

```diff
@@ -389,6 +390,37 @@ that will be used to call Lambda's [updateFunctionConfiguration](
 You can use this to modify the Lambda's timeout, change the memory size, or
 add your own environment variables. All sub keys are optional.
 
+#### DynamoDB TTL (optional)
+
+Litexa has limited support for setting a [Time To Live (TTL)](
+  https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html)
+field with each database entry. This may be useful for limiting your table
+data to active users by automatically deleting old user data. To set a Time To
+Live specification, add the `dynamoDbConfiguration` object to your Litexa config:
+
+```javascript
+{
+  deployments: {
+    development: {
+      module: "@litexa/deploy-aws",
+      askProfile: "suncoast",
+      awsProfile: "prototyping",
+      dynamoDbConfiguration: {
+        timeToLive: {
+          AttributeName: "timeToLive", // needs to match the attribute name specified when enabling TTL
+          secondsToLive: 2592000 // specified in seconds (would wipe data if not updated in 30 days)
+        }
+      }
+    }
+  }
+}
+```
+
+At this time, Litexa does not create or update your table with your TTL
+settings. You will need to [manually activate the TTL attribute for your table
+in the AWS console or AWS CLI](
+  https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/time-to-live-ttl-how-to.html).
```

Comment:
Couple things:
1) What happens when TTL isn't enabled and the skill is deployed with the above configuration - nothing?
2) Should specify that this step needs to be done after the skill is deployed (since that creates the table). Maybe something like:

:::warning
For your TTL settings to start working for skill data, you will need to [enable TTL](
  https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/time-to-live-ttl-how-to.html) on your Litexa skill table. Your skill table name can be found in the `artifacts.json` file as part of the `dynamoDBARN`, once the skill has been deployed.
```

Reply:
1.) yea, nothing would happen. It's just a field your skill would write each time it was invoked.